api/app/src/main/kotlin/packit/service/OutpackServerClient.kt

```diff
@@ -53,47 +87,22 @@ class OutpackServerClient(appConfig: AppConfig) : OutpackServer
         return response.body!! to response.headers
     }
 
-    private inline fun <reified T> getEndpoint(urlFragment: String): T
+    override fun getChecksum(): String
     {
-        val url = "$baseUrl/$urlFragment"
```

Comment:
there was some duplicated and superfluous code in this class, so have also just tidied this up